meta-xt-control-domain/recipes-guest/doma/u-boot-android-bazel_1.0.0.bb

```diff
@@ -0,0 +1,75 @@
+SUMMARY = "Deployment of Android U-Boot, which is built by the Bazel build system."
+DESCRIPTION = "Android U-Boot is a standard tool to boot Android."
+
+inherit base
+
+# PARAMETERS
+
+UBOOT_REPO_GIT_URL ?= "github.com/xen-troops/android_u-boot_manifest"
+
+# TBD. Find a way to form this variable automatically from the
+# "UBOOT_REPO_GIT_URL" parameter
+UBOOT_REPO_GIT_URL_DOTTED ?= "github.com.xen-troops.android_u-boot_manifest"
+
+UBOOT_REPO_DOWNLOAD_PROTOCOL ?= "https"
+UBOOT_REPO_GIT_BRANCH ?= "xenvm-trout-main"
+UBOOT_REPO_MANIFEST ?= "default.xml"
+U_BOOT_BUILD_TARGET ?= "xen-guest-android-virtio_aarch64"
+
+# IMPLEMENTATION
+
+UBOOT_DOWNLOAD_REPO_DIR = "${DL_DIR}/repo/${UBOOT_REPO_GIT_URL_DOTTED}/${UBOOT_REPO_MANIFEST}"
+UBOOT_LOCAL_REPO_RELATIVE_PATH = "repo/${UBOOT_REPO_GIT_URL_DOTTED}/${UBOOT_REPO_MANIFEST}"
+UBOOT_LOCAL_REPO_DIR = "${WORKDIR}/${UBOOT_LOCAL_REPO_RELATIVE_PATH}"
+UBOOT_BUILD_DIR = "${B}/${UBOOT_REPO_GIT_URL_DOTTED}/${UBOOT_REPO_MANIFEST}/${U_BOOT_BUILD_TARGET}"
+
+SRC_URI = "\
+           repo://${UBOOT_REPO_GIT_URL};protocol=${UBOOT_REPO_DOWNLOAD_PROTOCOL};branch=${UBOOT_REPO_GIT_BRANCH};manifest=${UBOOT_REPO_MANIFEST}  \
+           "
+
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/${UBOOT_LOCAL_REPO_RELATIVE_PATH}/repo/u-boot/Licenses/README;md5=2ca5f2c35c8cc335f0a19756634782f1"
+
+FILES:${PN} = "\
+               ${libdir}/xen/boot/u-boot-doma \
+               "
+
+do_unpack() {
```

Comment:
Why is wrong with default repo unpacker? Why you are not using it?

Reply:
Hi @lorc,

Refactored. Still works.

Done.